server/pkg/sqlutil/tenant.go

```diff
@@ -0,0 +1,58 @@
+package sqlutil
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/kubeshop/tracetest/server/http/middleware"
+)
+
+func Tenant(ctx context.Context, query string, params ...any) (string, []any) {
```

Comment:
From here we can grab add the tenant query to the operation and grab the tenant id from the context